lib/main.dart

```diff
@@ -38,9 +39,17 @@ Future<void> runTmail() async {
   runApp(const TMailApp());
 }
 
-class TMailApp extends StatelessWidget {
+class TMailApp extends StatefulWidget {
```

Comment:
What is the purpose of `StatefulWidget` for `TMailApp`?

Reply:
Manage object of TwakeAppManager